### DISCUSSION TITLE
Can I call the PayPal client to initiate a subscription? On iOS.

### DISCUSSION THREAD

**Discussion by PreSwift** [paypal/paypal-ios#199] [Q&A] [answered]
On iOS，Now I have obtained the Plan, and I want to realize subscription by calling PayPal client.  Can it be achieved?

**Comment by scannillo** [comment, marked as answer]
The SDK currently does not support subscriptions/recurring billing, though it is on our roadmap to add at a later date. Timeline for this feature work has not yet been dedicated. Please open a [feature request](https://github.com/paypal/paypal-ios/issues/new/choose) so we can gauge community interest on adding support.
